src/wdio/services/TerraCommands/axe-command.js

```diff
@@ -30,7 +30,18 @@ const injectAxe = (axeOptions) => {
 *
 * @returns {Object} axeResults - the axe results as list of passes, violations, incomplete and inapplicable.
 */
-const runAxeTest = (rules) => {
+const runAxeTest = (axeRules) => {
+  const globalAxeRules = (browser.options.axe && browser.options.axe.options) ? browser.options.axe.options.rules : undefined;
+  // Converting `globalAxeRules` from array of rules object to object of rules.
```

Comment:
I am a bit confused — what stops us from providing the rules in the format below in _wdio_ config? Since toolkit dictates how the rules should look like (imo) and is responsible for massaging them and running the `axe` command against them, I'd expect it to work and further avoid the overhead of maintaining two different formats and having to do this for every accessibility test run. 

```
axe: {
   inject: true, 
   options: {
       rules: {
         'color-contrast': { enabled: true },
         'valid-lang': { enabled: false }
      }
   }
}
```

Reply:
All global axe rules specified in the custom wdio config are injected using the `axe.configure` as seen [here](https://github.com/cerner/terra-toolkit/blob/master/src/wdio/services/TerraCommands/axe-command.js#L64).
So, the above mentioned format will not be supported as per the [documentation](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#api-name-axeconfigure) and only accepts rules as an array of objects in the specified format.